lib/xeroizer/models/payroll/leave_type.rb

```diff
@@ -10,16 +10,18 @@ class LeaveType < PayrollBase
 
         string        :name
         string        :type_of_units
-        boolean       :is_paid_leave
+        boolean       :is_paid_leave # if paid leave this will need :leave_category_code
         boolean       :show_on_payslip
 
         guid          :leave_type_id
         decimal       :normal_entitlement
         decimal       :leave_loading_rate
+        string        :leave_category_code, api_name: 'LeaveCategoryCode' # https://developer.xero.com/documentation/api/payrollau/payitems#elements-for-leavetypes
+        boolean       :SGC_exempt, api_name: 'SGCExempt' # list of what is exempt here: https://developer.xero.com/documentation/api/payrollau/types-and-codes#leavecategory
 
         datetime_utc  :updated_date_utc, :api_name => 'UpdatedDateUTC'
 
-        validates_presence_of :name, :type_of_units, :is_paid_leave, :show_on_payslip
+        validates_presence_of :name, :type_of_units, :is_paid_leave, :show_on_payslip, :SGC_exempt
```

Comment:
I don't think this is required either?  So this will end up failing if it's not set in Xero and we try posting back

![image](https://user-images.githubusercontent.com/8640184/203227198-0a9fcabd-07cf-462a-88ff-efc108fd686e.png)